src/mover.py

```diff
@@ -6177,7 +6180,10 @@ def connect_client(self):
 
             #self.listen_socket.listen(1)
             if (not self.method) or (self.method and self.method != 'read_next'):
```

Comment:
pre-existing code, but I think the following works : 
```
if self.method != 'read_next':
```

None != 'read_next' evaluates to True

Reply:
I do not understand.  Suggest to leave as is because this was not changed.